parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -209,6 +209,44 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 				pallet_assets::Call::touch { .. } |
 				pallet_assets::Call::refund { .. },
 			) |
+			RuntimeCall::Nfts(
+				pallet_nfts::Call::create { .. } |
+				pallet_nfts::Call::force_create { .. } |
+				pallet_nfts::Call::destroy { .. } |
+				pallet_nfts::Call::mint { .. } |
+				pallet_nfts::Call::force_mint { .. } |
+				pallet_nfts::Call::burn { .. } |
+				pallet_nfts::Call::transfer { .. } |
+				pallet_nfts::Call::lock_item_transfer { .. } |
+				pallet_nfts::Call::unlock_item_transfer { .. } |
+				pallet_nfts::Call::lock_collection { .. } |
+				pallet_nfts::Call::transfer_ownership { .. } |
+				pallet_nfts::Call::set_team { .. } |
+				pallet_nfts::Call::force_collection_owner { .. } |
+				pallet_nfts::Call::force_collection_config { .. } |
+				pallet_nfts::Call::approve_transfer { .. } |
+				pallet_nfts::Call::cancel_approval { .. } |
+				pallet_nfts::Call::clear_all_transfer_approvals { .. } |
+				pallet_nfts::Call::lock_item_properties { .. } |
+				pallet_nfts::Call::set_attribute { .. } |
+				pallet_nfts::Call::force_set_attribute { .. } |
+				pallet_nfts::Call::clear_attribute { .. } |
+				pallet_nfts::Call::approve_item_attributes { .. } |
+				pallet_nfts::Call::cancel_item_attributes_approval { .. } |
+				pallet_nfts::Call::set_metadata { .. } |
+				pallet_nfts::Call::clear_metadata { .. } |
+				pallet_nfts::Call::set_collection_metadata { .. } |
+				pallet_nfts::Call::clear_collection_metadata { .. } |
```

Comment:
Do these have bounds on their encoded size?

Reply:
yep, none of those methods accept an unbounded vecs, only the bounded ones